openedx/core/djangoapps/notifications/models.py

```diff
@@ -5,6 +5,19 @@
 from django.db import models
 from model_utils.models import TimeStampedModel
 
+from opaque_keys.edx.django.models import CourseKeyField
+
+
+NOTIFICATION_PREFERENCE_CONFIG_DEFAULT = {
```

Comment:
This is the default configuration? Will we update this when a new notification type is added?

Reply:
Also add a comment here that every time you update this configuration you also need to update the version

Reply:
@saadyousafarbi I do not think this should be named default. This will change once we update the configuration.